src/foam/u2/grid/testdata/HeroModels.js

```diff
@@ -0,0 +1,213 @@
+foam.CLASS({
+    refines: 'foam.core.Property',
+    properties: [
+        {
+           name: 'referenceDAOKey',
```

Comment:
I wrote a referenceDAO which is being pushed like, now. 
Which gets the object property from the Id property. 
I.e., gets the organizaiton property when organizaitonId is given.